assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -100,6 +100,29 @@ const normalizeModules = memize( ( serverDefinitions, clientDefinitions ) => {
 		}, {} );
 } );
 
+/**
+ * Memoized function to build an object mapping recoverable module slugs to their corresponding
+ * module objects.
```

Comment:
This should trigger our `sitekit/jsdoc-third-person` rule but it looks like it isn't run on doc blocks for functions defined as variables.

Doc blocks aren't needed for such internal functions, but it doesn't hurt to keep; let's update it to satisfy the check though for consistency :)

```suggestion
 * Gets a memoized object mapping recoverable module slugs to their corresponding
 * module objects.
```